src/build/stable/commands/compile/wasm_binary/rust/stable_canister_template/src/execute_method_js.rs

```diff
@@ -29,14 +29,24 @@ fn execute_method_js_with_result(
     pass_arg_data: bool,
 ) -> Result<(), Box<dyn Error>> {
     quickjs_with_ctx(|ctx| {
-        let callbacks: Object = ctx
+        let exported_canister_class_instance: Object = ctx
```

Comment:
And this is exactly the same except we are getting the exported canister class instance and *then* getting the callbacks off of that right?